styles.css

```diff
@@ -0,0 +1,85 @@
+:root {
+  --black-color: #1a2236;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  /* width: 320px; */
+  background: url("images/image_geometry_1.png");
+  background-repeat: no-repeat;
+  position: relative;
+}
+
+.mobile-nav {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 16px;
+  padding-top: 19px;
+
+  /* left: 284px;
+  top: 31px; */
+}
+
+.name-tag {
+  width: 220px;
+  height: 100px;
+  margin-top: 104px;
+  margin-bottom: 0%;
+  font-family: "Playfair Display", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+  margin-left: 35px;
+  align-self: flex-start;
```

Comment:
- [ ] Kindly be noticed that it is better to remove the `margin-left` and `align-self` to make the `h1` tag in the center of the page.

Reply:
Hello, @ShahierNashaat Thanks a lot for your feedback. I put the margin-left and align-self on the h1 tag so that it aligns with the div just as It was in the template. But if I remove them, It no longer aligns like it was in the template. So should I remove them or not?
![Screenshot 2022-08-18 155925](https://user-images.githubusercontent.com/102699958/185441270-262e78c8-3ab6-4d43-a9a9-3b6ee8cb4a1a.png)
![Screenshot 2022-08-18 155649](https://user-images.githubusercontent.com/102699958/185441348-ff3e2d0a-4073-406e-97e3-0f1e9b676deb.png)
.